_posts/2024-05-27-gsoc-outreachy-students.md

```diff
@@ -0,0 +1,86 @@
+
+# GSoC
+
+<a href="https://summerofcode.withgoogle.com/"> <img
+    src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+    title="Google Summer of Code" alt="Google Summer of Code"
+    style="display: inline; float: right; height: 4em; margin: 0
+    0.5em" /></a>
+
+## Lawson Woods: [Enhancing the Interoperability and Efficiency of the Zarrtraj MDAKit](https://summerofcode.withgoogle.com/programs/2024/projects/BYYAE9MR)
+
+<img
+src="https://avatars.githubusercontent.com/ljwoods2"
+title="Lawson Woods" alt="Lawson Woods"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Streaming trajectory data directly from cloud storage will allow researchers to share their trajectories more easily than ever before, however, rewriting existing trajectory file readers for cloud-streaming would be a slow and complex process. [zarrtraj](https://github.com/Becksteinlab/zarrtraj), a trajectory file format based on [Zarr](https://zarr.readthedocs.io/en/stable/) storage, presents an alternative which natively supports AWS S3, Google Cloud Buckets, and Azure Blob Storage in addition to reading from a local directory without much added complexity. Lawson's GSoC 2024 project will be to continue enhancing the ease-of-use, robustness, and efficiency of this format.
+
+Lawson is a computer science student at Arizona State University (ASU) graduating in December 2024. You can find Lawson on GitHub as [@ljwoods2](https://github.com/ljwoods2) and on [LinkedIn](https://www.linkedin.com/in/lawson-woods/). To see updates on the `zarrtraj` project as well as his other projects you can check out [his blog](https://ljwoods2.github.io/).
+
+## Valerij Talagayev: [2D Visualization for Small Molecules](https://summerofcode.withgoogle.com/programs/2024/projects/sfy3kuqc)
+
+<img
+src="https://avatars.githubusercontent.com/talagayev"
+title="Valerij Talagayev" alt="Valerij Talagayev"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+2D Visualization of small molecules is quite a helpful way to recognize the different parts of the small molecule. The main objective of Valerij's GSoC project will revolve around the 2D visualization of small molecules through the application of the MDAnalysis RDKit converter, which would allow to convert ligands into RDKit objects and thus allow the user to look at the ligand in his MDAnalysis Universe in an easy to use interactive display in [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/).
```

Comment:
suggestion: I think the first sentence is a bit redundant with the rest. I've also added a few words to make the link with RDKit clearer (i.e. we use it for its drawing code) and also how these drawing can be useful from an MD perspective:
```suggestion
The main objective of Valerij's GSoC project will revolve around the visualization of small molecules in 2D through the application of the MDAnalysis RDKit converter, which allows converting MDAnalysis atomgroups into RDKit objects and benefit from its powerful drawing code to display small molecules. These can then be annotated at the atom level with information associated with the MDAnalysis Universe or arbitrary data in an easy to use interactive display in [Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/).
```

Reply:
Pinging @talagayev to confirm approval of the change

Reply:
looks good to me, thanks @jennaswa @cbouy :)